src/parallel/ScatterSplitter.h

```diff
@@ -56,7 +56,7 @@ class ScatterSplitter :  public SimpSMTSolver, public Splitter {
 protected:
     bool     scatterLevel();                                                  // Are we currently on a scatter level.
     opensmt::pair<SplitData,lbool> createSplitAndBlockAssumptions();          // Create a split formula and place it to the splits vector.
-    bool     excludeAssumptions(vec<Lit> const & neg_constrs);                // Add a clause to the database and propagate
+    bool     excludeAssumptions(vec<Lit> && neg_constrs);                     // Add a clause to the database and propagate
```

Comment:
Why not const anymore?

Reply:
To avoid extra copy. It just passes the vector to `addOriginalClause`.

Reply:
Ok, makes sense, one can argue that const is good because it guarantees that excludeAssumptions won't change anything => it will be more safe...

But tbh I am far away from being c++ purist with strong opinions 😄

Reply:
rvalue (`&&`) means the caller no longer care what happens to the vector, the callee can do whatever they want to do with it.